engine/src/gfx/cockpit.cpp

```diff
@@ -2140,11 +2139,14 @@ void GameCockpit::ScrollAllVDU(int howmuch) {
 }
 
 void GameCockpit::SetStaticAnimation() {
-    const string comm_static = configuration().graphics.comm_static;
-    static Animation Statuc(comm_static.c_str());
+    const std::string& comm_static = configuration().graphics.comm_static;
+    static boost::optional<Animation> Statuc{};
```

Comment:
Surely this is a typo and they meant status?!

Reply:
No, they meant `static`. Of course, that's a keyword, so they intentionally misspelled it. At least as far as I can tell.

Reply:
<sigh>
<heavy sigh>
<SCREEEAAM>
`
static boost::optional<Animation> Static_animation{};`

Reply:
Good suggestion. Done